content/en/docs/collector/deployment/agent.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Agent
+description:
+  Why and how to send signals to collectors and from there to backends
+weight: 2
+---
+
+The agent collector deployment pattern consists of
+applications—[instrumented][instrumentation] with an OpenTelemetry SDK using
+[OpenTelemetry protocol (OTLP)][otlp]—or other collectors (using the OTLP
```

Comment:
```suggestion
applications — [instrumented][instrumentation] with an OpenTelemetry SDK using
[OpenTelemetry protocol (OTLP)][otlp] — or other collectors (using the OTLP
```

my feeling is that there should be spaces between words and dashes, but I might be wrong, @chalin wdyt?